src/m2n/config/M2NConfiguration.cpp

```diff
@@ -67,12 +67,25 @@ M2NConfiguration::M2NConfiguration(xml::XMLTag &parent)
     attrExchangeDirectory.setDocumentation(doc);
     attrExchangeDirectory.setDefaultValue("");
     tag.addAttribute(attrExchangeDirectory);
+    tags.push_back(tag);
+  }
+  {
+    XMLTag tag(*this, "mpi-singleports", occ, TAG);
+    doc = "Communication via MPI with startup in separated communication spaces, using a single communicator";
+    tag.setDocumentation(doc);
 
+    XMLAttribute<std::string> attrExchangeDirectory(ATTR_EXCHANGE_DIRECTORY);
+    doc = "Directory where connection information is exchanged. By default, the ";
+    doc += "directory of startup is chosen, and both solvers have to be started ";
+    doc += "in the same directory.";
+    attrExchangeDirectory.setDocumentation(doc);
+    attrExchangeDirectory.setDefaultValue("");
+    tag.addAttribute(attrExchangeDirectory);
     tags.push_back(tag);
   }
-  
+
   {
-    XMLTag tag(*this, VALUE_MPI_SINGLE, occ, TAG);
```

Comment:
It's like that (values instead of variables) at a couple of other configurations. IMHO it's easier to read and easier to write and extend. If changes are necessary, it's usually only at two places. Furthermore if values are changes, the variable names (which basically are similiar to the values) become inconsistent, which does much more harm to comprehensibility than inconsistency among different configurations of different packages.

Reply:
OK. I did not know that it is already like that in other places. I agree with the easier-to-read.